larpandoracontent/LArHelpers/LArHitWidthHelper.h

```diff
@@ -281,10 +281,31 @@ class LArHitWidthHelper
      *  @param  constituentHitVector the input vector of contituent hits
      *  @param  lowerXCoordinate the lower x extremal point
      *  @param  higherXCoordinate the higher x extremal point
-     *
      */
     static void GetExtremalCoordinatesX(const ConstituentHitVector &constituentHitVector, pandora::CartesianVector &lowerXCoordinate,
         pandora::CartesianVector &higherXCoordinate);
+
+    /**
+     *  @brief  Consider the hit width to find the closest position of a calo hit to a specified line
+     *
+     *  @param  lineStart the start position of the line 
+     *  @param  lineDirection the direction of the line
+     *  @param  pCaloHit the input calo hit
+     *
+     *  @return  CartesianVector the closest position
```

Comment:
No need for type in return comment.

Reply:
ah yes, i'll change the rest of this .h file too since i seemed to have made this mistake in the earlier release